common/components/CommentsCard/index.tsx

```diff
@@ -44,24 +44,30 @@ const CommentsCard: React.FC<Props> = ({ taskId, loading = false }) => {
     })
     setInput('')
   }
+
   // const bottomRef = useRef(null)
   // useEffect(() => {
-  //   bottomRef.current?.scrollIntoView({ behavior: 'smooth' })
-  // }, [input])
+  //   bottomRef.current?.scrollIntoView()
+  // }, [task])
+  const bottomRef = useRef(null)
+  useEffect(() => {
+    bottomRef.current?.scrollIntoView()
+  }, [task])
 
   return (
-    <Card loading={loading} className={s.Card} title="Коментарі">
-      <List
-        className={s.List}
-        dataSource={task?.data?.comment}
-        renderItem={(item, index) => (
-          <List.Item key={index}>
-            <Comment comment={item} taskId={task?.data?._id} />
-            {/* <div ref={bottomRef} /> */}
-          </List.Item>
-        )}
-      />
-
+    <div className={s.CardDiv}>
+      <Card loading={loading} className={s.Card} title="Коментарі">
+        <List
+          className={s.List}
+          dataSource={task?.data?.comment}
+          renderItem={(item, index) => (
+            <List.Item key={index}>
+              <Comment comment={item} taskId={task?.data?._id} />
+              <div ref={bottomRef} />
+            </List.Item>
```

Comment:
This code fragment:
```
<List.Item key={index}>
    <Comment comment={item} taskId={task?.data?._id} />
    <div ref={bottomRef} />
</List.Item>
```
is equal to the next one:
```
<List.Item key={index}>
    <Comment ref={bottomRef} comment={item} taskId={task?.data?._id} />
</List.Item>
```
or
```
<List.Item key={index} ref={bottomRef} >
    <Comment comment={item} taskId={task?.data?._id} />
</List.Item>
```
so this `<div>` is unnecessary.